### PR TITLE
Native YAML - windows

### PR DESCRIPTION
**Pull request by Fale**
##### ISSUE TYPE
 - Docs Pull Request

##### COMPONENT NAME
windows/*

##### ANSIBLE VERSION
```
2.2
```

##### SUMMARY
Make YAML code use the native style

@gundalow

### CODE REVIEW

**Comment by gundalow** [comment]
@nitzmahone Can you please check the use of `\` v `/` in the paths. Thanks

File: windows/win_updates.py [hunk 1]
```diff
@@ -69,15 +69,22 @@
...
+- win_updates:
+    category_names: SecurityUpdates
 
-    # Search-only, return list of found updates (if any), log to c:\ansible_wu.txt
-    win_updates: category_names=SecurityUpdates state=searched log_path=c:/ansible_wu.txt
+# Search-only, return list of found updates (if any), log to c:\ansible_wu.txt
+- win_updates:
+    category_names: SecurityUpdates
+    state: searched
+    log_path: 'c:/ansible_wu.txt'
```
Comment (nitzmahone) [inline windows/win_updates.py]: If we're going for consistency with some upcoming docs changes, this should be unquoted and backlashed normally, ie `log_path: c:\ansible_wu.txt`.

**Comment by gregdek** [comment]
Thanks @Fale. To the current maintainers, @SamLiu79, @nitzmahone, @petemounce please review according to guidelines (http://docs.ansible.com/ansible/developing_modules.html#module-checklist) and comment with text 'shipit', 'needs_revision' or 'close_me' as appropriate.

[This message brought to you by your friendly Ansibull-bot.]

File: windows/win_updates.py [hunk 2]
```diff
@@ -80,11 +80,11 @@
 - win_updates:
     category_names: SecurityUpdates
 
-# Search-only, return list of found updates (if any), log to c:\ansible_wu.txt
+# Search-only, return list of found updates (if any), log to c:\\ansible_wu.txt
 - win_updates:
     category_names: SecurityUpdates
     state: searched
-    log_path: 'c:/ansible_wu.txt'
+    log_path: c:\\ansible_wu.txt
```
Comment (nitzmahone) [inline windows/win_updates.py]: Nope, just a normal backslash. Unquoted YAML strings don't need to be escaped.

Comment (nitzmahone) [reply windows/win_updates.py]: http://yaml.org/spec/1.1/#\ escaping in double-quoted style/

**Comment by nitzmahone** [comment]
Looks good- thanks!

**Pull request merged**